frontend/taipy-gui/src/components/Taipy/DateRange.tsx

```diff
@@ -6,8 +6,8 @@
  *
  *        http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
- * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
```

Comment:
please do not modify head comment as it is generated and eventually replaced automatically.